xbmc/cores/RetroPlayer/process/gbm/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 set(SOURCES RPProcessInfoGbm.cpp)
-
```

Comment:
Might as well make it 2 lines deleted :)

Reply:
I'm not sure what you mean?